contracts/proxy/ERC1967/ERC1967Upgrade.sol

```diff
@@ -134,30 +153,31 @@ abstract contract ERC1967Upgrade is IERC1967 {
         if (newAdmin == address(0)) {
             revert ERC1967InvalidAdmin(address(0));
         }
-        StorageSlot.getAddressSlot(_ADMIN_SLOT).value = newAdmin;
+        StorageSlot.getAddressSlot(ADMIN_SLOT).value = newAdmin;
     }
 
     /**
      * @dev Changes the admin of the proxy.
      *
      * Emits an {AdminChanged} event.
```

Comment:
Perhaps we want to point to `IERC1967` events reference with:

```suggestion
     * Emits an {IERC1967-AdminChanged} event.
```

Apply for the rest of the events. Non-blocking either.